app/email_utils.py

```diff
@@ -1395,7 +1409,9 @@ def generate_verp_email(
     ).lower()
 
 
-def get_verp_info_from_email(email: str) -> Optional[Tuple[VerpType, int]]:
+def get_verp_info_from_email(
+    email: str, validate_time: bool = True
```

Comment:
Do we have a caller that uses the `validate_time` param?

Reply:
removed